docs/android/tips/dependencies.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Android dependencies"
+sidebar_label: "Dependencies"
+---
+
+## Version catalog
```

Comment:
Would this be a good place to mention not to use an alpha library unless a specific feature is required? We do have some alphas but we dont go out of our way to use every alpha....only when we need it or doing something new.